apps/playground/src/app/modules/components/search-bar-module/search-bar-module.component.html

```diff
@@ -9,168 +9,196 @@ <h2>DsfrSearchBarComponent</h2>
     It can be used on its own :
   </p>
   <ul>
-      <li>as a simple input, to retrieve the search query entered by the user after they click on the search button, hook onto the <code>searchQuerySubmitted</code> Output property</li>
-      <li>as an input with autocompletion, to retrieve the search result chosen by the user, hook onto the <code>searchResultSelected</code> Output property</li>
+    <li>as a simple input, to retrieve the search query entered by the user after they click on the search button, hook
+      onto the <code>searchQuerySubmitted</code> Output property
+    </li>
+    <li>as an input with autocompletion, to retrieve the search result chosen by the user, hook onto the <code>searchResultSelected</code>
+      Output property
+    </li>
   </ul>
   <p>
     It can also be used as a <code>FormControl</code>, with the ReactiveForms API.
   </p>
```

Comment:
```html
<p>
    It can also be used as a <code>FormControl</code>, with the ReactiveForms API. In that case, you should probably : 
  </p>
<ul>
    <li>put the <code>hasSearchButton</code> Input property to <code>false</code>. That way, no search button will be present in the DOM and the responsibility to submit the value will be for the parent form's submit button</li>
    <li>pass a value to the <code>label</code> and <code>hint</code> Input properties, as this time, they will be displayed</li>
</ul>
```